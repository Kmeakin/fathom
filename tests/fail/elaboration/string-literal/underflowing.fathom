//~ exit-code = 1

"oops" : U64
