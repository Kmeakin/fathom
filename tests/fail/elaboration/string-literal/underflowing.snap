stdout = ''
stderr = '''
error: mismatched number of bytes in string literal
  ┌─ tests/fail/elaboration/string-literal/underflowing.fathom:3:1
  │
3 │ "oops" : U64
  │ ^^^^^^ invalid string literal
  │
  = expected byte length 8
  =    found byte length 4

'''
